<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VNAON2C" failures="1" tests="5" name="com.qa.adminTest.ProductTest" time="73.220" errors="1" timestamp="2024-01-10T19:59:48 IST" skipped="0">
  <testcase classname="com.qa.adminTest.ProductTest" name="addNewProduct" time="6.188"/>
  <system-out/>
  <testcase classname="com.qa.adminTest.ProductTest" name="updateProduct" time="5.692"/>
  <system-out/>
  <testcase classname="com.qa.adminTest.ProductTest" name="deleteProduct" time="5.253">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.qa.adminTest.ProductTest.deleteProduct(ProductTest.java:123)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- deleteProduct -->
  <system-out/>
  <testcase classname="com.qa.adminTest.ProductTest" name="addProductUnderCatgory" time="7.293"/>
  <system-out/>
  <testcase classname="com.qa.adminTest.ProductTest" name="updateProductInategory" time="48.794">
    <error message="Error {
  message=&#039;Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator(&quot;input[name=\&quot;products_name[1]\&quot;]&quot;)
============================================================
  name=&#039;TimeoutError
  stack=&#039;TimeoutError: Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator(&quot;input[name=\&quot;products_name[1]\&quot;]&quot;)
============================================================
    at ProgressController.run (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\progress.js:88:26)
    at Frame.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\frames.js:1039:23)
    at FrameDispatcher.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\frameDispatcher.js:166:30)
    at DispatcherConnection.dispatch (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\dispatcher.js:346:46)
}" type="com.microsoft.playwright.TimeoutError">
      <![CDATA[com.microsoft.playwright.TimeoutError: Error {
  message='Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator("input[name=\"products_name[1]\"]")
============================================================
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator("input[name=\"products_name[1]\"]")
============================================================
    at ProgressController.run (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\progress.js:88:26)
    at Frame.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\frames.js:1039:23)
    at FrameDispatcher.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\frameDispatcher.js:166:30)
    at DispatcherConnection.dispatch (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\dispatcher.js:346:46)
}
at com.microsoft.playwright.impl.WaitableResult.get(WaitableResult.java:52)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:120)
at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:126)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:106)
at com.microsoft.playwright.impl.FrameImpl.fillImpl(FrameImpl.java:335)
at com.microsoft.playwright.impl.PageImpl.lambda$fill$27(PageImpl.java:656)
at com.microsoft.playwright.impl.LoggingSupport.lambda$withLogging$0(LoggingSupport.java:36)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:89)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:35)
at com.microsoft.playwright.impl.PageImpl.fill(PageImpl.java:655)
at com.microsoft.playwright.Page.fill(Page.java:4545)
at com.qa.adminPages.ProductPage.updateProduct(ProductPage.java:78)
at com.qa.adminTest.ProductTest.updateProductInategory(ProductTest.java:183)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.microsoft.playwright.TimeoutError: Error {
  message='Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator("input[name=\"products_name[1]\"]")
============================================================
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for locator("input[name=\"products_name[1]\"]")
============================================================
    at ProgressController.run (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\progress.js:88:26)
    at Frame.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\frames.js:1039:23)
    at FrameDispatcher.fill (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\frameDispatcher.js:166:30)
    at DispatcherConnection.dispatch (C:\Users\DELL\AppData\Local\Temp\playwright-java-1753537099222818813\package\lib\server\dispatchers\dispatcher.js:346:46)
}
at com.microsoft.playwright.impl.Connection.dispatch(Connection.java:224)
at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:206)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:118)
... 34 more
]]>
    </error>
  </testcase> <!-- updateProductInategory -->
  <system-out/>
</testsuite> <!-- com.qa.adminTest.ProductTest -->
